docs/upgrade.html

```diff
@@ -48,7 +48,6 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
             Before updating your Kafka cluster which build on Zookeeper to Kafka 4.0, you must understand what the updates and changes 
             are between Zookeeper mode and Kraft mode. See the following page to learn about some of the significant changes in 
             latest Kafka releases.
```

Comment:
"which **build** on" should be "which was built on" or "built". Also, I'd like to suggest more concise wording and add a hyperlink. What do you think?

```suggestion
            Before updating your Kafka cluster built on Zookeeper to Kafka 4.0, you must understand the updates and changes
            between Zookeeper mode and Kraft mode. See <a href="#upgrade-significant-changes">Significant Changes</a> in latest Kafka releases for more information.
```
Example:
![image](https://github.com/user-attachments/assets/a1274554-6a38-4d72-b599-9c12c6ae1d75)

Reply:
Hi @mingdaoy 
The rephase looks good to me.
But the anchor link you suggested does not seem to work; maybe I missed something?